src/deepsparse/utils/onnx.py

```diff
@@ -129,7 +129,7 @@ def model_to_path(model: Union[str, Model, File]) -> str:
 
     if Model is not object and isinstance(model, Model):
         # trigger download and unzipping of deployment directory if not cached
-        model.deployment_directory_path
+        model.deployment.download()
```

Comment:
Why do we want download instead of path? Would this trigger redownload?

Reply:
Yeah, I agree